dragos-worldview-threat-intelligence/connector.py

```diff
@@ -6,10 +6,10 @@
 from connectors.core.connector import Connector, get_logger, ConnectorError
 from .operations import operations, _check_health
 
-logger = get_logger("dragos")
+logger = get_logger("dragos-worldview-threat-intelligence")
 
 
-class DragosConnector(Connector):
+class DragosWorldviewConnector(Connector):
```

Comment:
name can be "DragosWorldviewThreatIntel"

Reply:
Class name changed to "DragosWorldViewThreatIntel"